ansible/configs/base-infra/pre_software.yml

```diff
@@ -0,0 +1,93 @@
+---
+- name: Step 003 Pre Software
+  hosts: localhost
+  gather_facts: false
+  become: false
+  tags:
+    - step004
+    - pre_software
+
+  tasks:
+
+    - name: "Step 003 Pre Software"
+      ansible.builtin.debug:
+        msg: "Step 000 Pre Software"
+
+    - name: Set local ssh key
+      when: agd_set_env_authorized_key | default(true) | bool
+      ansible.builtin.import_role:
+        name: create_ssh_provision_key
+
+- name: Configure all hosts with Repositories, Common Files and Set environment key
+  hosts: all
+  become: true
+  gather_facts: false
+  tags:
```

Comment:
Can we get rid of tags?

Reply:
Let's postpone that until we have agreed a position on that. Many/most configs have them and they are "free"